lib/public/views/Logs/Create/TemplatedLogCreationModel.js

```diff
@@ -27,15 +27,16 @@ import { RcDailyMeetingTemplate } from './templates/RcDailyMeetingTemplate.js';
  * Return a new instance of log template for the given key
  *
  * @param {logTemplateKey} key the template key
+ * @param {LogCreationTemplateData} templateData the template data
```

Comment:
What is LogCreationTemplateData? 
I think here the best you can put is `object`